thirdparty/CMakeLists.txt

```diff
@@ -105,6 +105,14 @@ macro(build_faiss)
                  --without-cuda )
     endif ()
 
+    if (MSYS)
+        # patch for folder faiss/faiss, symlink not work for msys
+        execute_process(
+            COMMAND bash -c "test -f faiss/faiss/Index.h || (rm -fr faiss/faiss && cp -fr faiss faiss.tmp && mv faiss.tmp faiss/faiss)"
```

Comment:
for faiss-1.7.2, faiss/faiss is not a symbol link any more

Reply:
I'll remove it in the next rebased version.